### ISSUE TITLE
Error found

### ISSUE THREAD

**Issue by YJJ0829** [qq456cvb/Point-Transformers#48]
An error seems occured in ./models/Nico/model.py line 88 and line 89. d_model_g was mistakenly assigned the value of 'cfg.model.local_dim', while d_model_l was mistakenly assigned the value of 'cfg.model.glaobal_dim'

**Comment by qq456cvb** [comment]
Can you submit a pull request? Thank you for pointing this out!

**Comment by qq456cvb** [comment]
Good catch @YJJ0829. `d_model_l` and `d_model_g` were indeed read from `global_dim`/`local_dim` in the wrong order. Fixed on master in 5f6ef58. Since `global_dim` (512) and `local_dim` (256) differ, this actually changes the Nico architecture, so the old Nico numbers will shift after retraining.
